rocAL/rocAL/include/augmentations/color_augmentations/node_gamma.h

```diff
@@ -25,15 +25,12 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-
-class GammaNode : public Node
-{
+class GammaNode : public Node {
```

Comment:
This only contains formatting changes. Please check as it introduces one new file change in PR